docs/The-Free-Plan.md

```diff
@@ -0,0 +1,70 @@
+---
+---
```

Comment:
Oh also today we pushed the Jekyll plugin to add SEO tags. I updated the instructions [here](https://github.com/Expensify/App/tree/main/docs#add-meta-description-tags-for-seo) to include some SEO information in these lines.